modules/database.js

```diff
@@ -25,20 +25,28 @@ function initPlayersTable(db) {
     console.log("Created players table");
 
     console.log("Importing players from players.tsv");
+
+    const promisisArray = [];
```

Comment:
```suggestion
    const promisesArray = [];
```
don't forget to refactor

Reply:
oh hahahaha nice typo 😂 

thats gonna be fun for me, I branched this of a commit thats way down my own commit history so uh, guess I'll have to merge this on top of my other stuff to stay up to date 🤣  oh well shouldnt be too much of a problem xd